stixcore/time/datetime.py

```diff
@@ -446,6 +452,15 @@ def to_datetime(self):
         except TypeError:
             utc = Spice.instance.scet_to_datetime(self.to_string())
 
+        kernel_name = Spice.instance.meta_kernel_path[1].name.split('_')[4]
```

Comment:
We really shouldn't use any time conversations in the future compared to the meta kernel date as they are subject to change especially for science data!